lib/example_def.dart

```diff
@@ -1,47 +0,0 @@
-import 'package:flame/game.dart';
-import 'package:flame_example/examples/animations.dart';
-import 'package:flame_example/examples/composability.dart';
-
-import 'examples/aseprite.dart';
-import 'examples/debug.dart';
-
-class ExampleDef {
-  final String name;
-  final String description;
-  final Game Function() provider;
-
-  ExampleDef(this.name, this.description, this.provider);
-
-  static final all = [
-    ExampleDef(
-      'Animations',
-      'Showcase the SpriteAnimationComponent',
```

Comment:
I thought it would be good to show a brief desc into each example somewhere as well

Reply:
Sounds good, I think this could be a nice feature for Dashbook as well, I implement it there alongside the see code link